sql/migrations/20201010164136_world.sql

```diff
@@ -0,0 +1,41 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20201010164136');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20201010164136');
+-- Add your query below.
+
+
+-- add thornling ai
+UPDATE `creature_template` SET `ai_name` = 'EventAI' WHERE `entry` = 14362;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(1436201, 0, 59, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Thornling - Set React State'),
+(1436201, 0, 15, 23973, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Thornling - Cast Spell');
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (1436201, 14362, 0, 11, 0, 100, 0, 0, 0, 0, 0, 1436201, 0, 0, 'Thornling - Set ReactState Passive and Cast Root Self on Spawn');
+
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(1436202, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -19, 0, 0, 0, 0, 0, 0, 0, 0, 'Thornling - Say Text'),

```

Comment:
Negative broadcast text id? There is no such thing.

Reply:
If you are attempting to make it use "%s begins to grow!"
Then broadcast text id is 9374.

Reply:
Updated, thanks for reviewing.